v4l2py/device.py

```diff
@@ -1076,13 +1102,13 @@ def set_to_maximum(self):
 
 
 class IntegerControl(BaseNumericControl):
-    lower_bound = -2 ** 31
-    upper_bound = 2 ** 31
+    lower_bound = -(2**31)
+    upper_bound = 2**31
```

Comment:
isn't max int 32bits: `2**31 - 1` ?

Reply:
Indeed, thanks, good catch. Fixed in [d485978](https://github.com/tiagocoutinho/v4l2py/pull/42/commits/d485978202c0a20925690d1b9707a1351bf1e6e7).